daemon/cmld.c

```diff
@@ -234,9 +234,6 @@ cmld_container_stop_cb(container_t *container, container_callback_t *cb, void *d
 	/* unregister observer */
 	container_unregister_observer(container, cb);
 
-	audit_log_event(container_get_uuid(container), SSA, CMLD, CONTAINER_MGMT,
```

Comment:
what was the reason for removing this?

Reply:
i wrote a detailed commit message to clarify this. The whole commit is only about this line